todo/tests/testcontainers/mongo_container.py

```
@@ -0,0 +1,57 @@
+            "--quiet",
+            "--host",
+            "localhost",
+            "--port",
+            "27017",
+            "--eval",
+            f"rs.initiate({initiate_js})",
+        ]
+        exit_code, output = self.exec(cmd)
+        if exit_code != 0:
+            raise RuntimeError(
+                f"rs.initiate() failed (exit code {exit_code}):\n{output.decode('utf-8', errors='ignore')}"
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)